tools/topology/topology2/include/common/audio_format.conf

```diff
@@ -232,6 +232,7 @@ Class.Base."audio_format" {
 	out_fmt_cfg	"$[($out_channels | ($out_valid_bit_depth * 256)) | ($out_sample_type * 65536)]"
 
 	# math expression for computing input/put buffer sizes
-	ibs "$[($in_channels * ($in_rate / 1000)) * ($in_bit_depth / 8)]"
-	obs "$[($out_channels * ($out_rate / 1000)) * ($out_bit_depth / 8)]"
+	# for 22.05, 44.1, 88.2 and 176.4khz, we need to round it to ceiling value
+	ibs "$[($in_channels * ($[($in_rate + 950)] / 1000)) * ($in_bit_depth / 8)]"
+	obs "$[($out_channels * ($[($out_rate + 950)] / 1000)) * ($out_bit_depth / 8)]"
```

Comment:
Add of 950 is not enough to round up 11.025 kHz. + 999 should work if the arithmetic works like I'm guessing.

Reply:
o, so we can support 11025